pyre/extra_tests/parity_tests/text_signatures_python314.py

```
@@ -429,6 +455,94 @@ def check_descriptors(owner, expected):
 assert str(inspect.signature(enumerate.__class_getitem__)) == "(object, /)"
 
 
+class Sequence:
+    def __getitem__(self, index):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__getitem__`

(ANN204)